### PR TITLE
Update Simplified Chinese translation

### PR DESCRIPTION
**Pull request by UM-Li**

### CODE REVIEW

**Review by Abestanis — CHANGES_REQUESTED** [review]
Hey, thank you very much for these translations! ❤️ 

As a thanks, you can add your name to the [list of translators file](https://github.com/UM-Li/birdtray/blob/master/src/res/translators.md) if you want. That list will show up in the `About` dialog in Birdtray. You can find more information about it in [the `Translators file` section of our wiki](https://github.com/gyunaev/birdtray/wiki/Add-a-new-translation#translators-file).

I have requested some small changes, if you are ok with them please apply them to this pull request. It's not a problem if you don't have the time or are unsure how to do it, I will wait until next weekend and apply them for you if you couldn't do it by then.

@mattuylee if you have the time, could you please also approve the new translations?

File: src/translations/main_zh_cn.ts
Comment (mattuylee) [inline src/translations/main_zh_cn.ts:545]: personally, "变更" is better. but it's up to you~
```suggestion
        <translation>检测到变更时，等待</translation>
```

**Review by mattuylee — APPROVED** [review]
Thanks for your tweaks and updates! Translations look better now~

**Comment by UM-Li** [comment]
Hi @Abestanis, I've changed the parentheses and added the comments.  Please let me know if you've found further errors of this type.

**Comment by UM-Li** [comment]
Hi @mattuylee, thanks for proofreading 😀

#### 垃圾邮件

These folders are on Outlook and my own IMAP server (Dovecot-based with default settings on folder names) respectively.  The shorter version "垃圾" appears to be Thunderbird's naming convention for the default Junk folder, so I made this change.  But there could be service providers who name Junk folders in their own way I suppose.

folders

#### 检测到改变时

I agree with this one; have changed "改变" to "变更".

**Review by Abestanis — APPROVED** [review]
Thanks a lot, looks good from my side.

@mattuylee Thanks for looking at this so quickly! I'll let you decide if we should keep 垃圾 over 垃圾邮件. Once you have decided this is ready to be merged.

**Comment by Abestanis** [comment]
I'm going to go ahead and merge this, thank you very much @UM-Li!

@mattuylee If you want to change the translation for `Junk` back, feel free to open another pull request.

**Pull request merged**
